includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -554,7 +554,7 @@ private function get_average_progress_for_courses_table( $course_id ) {
 			// divided by the total possible lessons completed.
 			$average_course_progress_value = $completed_count / ( $course_students_count * $total_lessons ) * 100;
 			$average_course_progress       = esc_html(
-				sprintf( '%d%%', $average_course_progress_value )
+				sprintf( '%d%%', ceil( $average_course_progress_value ) )
```

Comment:
You've used `ceil` here but it should be `round`.

Reply:
Hey Donna this is old code. I've pushed this commit yesterday morning (my morning): https://github.com/Automattic/sensei/pull/4917/commits/a8a6df0620f73b8efa8f98a8cacd3bd990a200d3. Not sure, maybe you haven't pulled the latest changes.